python/tk_multi_workfiles/actions/action.py

```diff
@@ -1,44 +1,85 @@
 # Copyright (c) 2015 Shotgun Software Inc.
-# 
+#
 # CONFIDENTIAL AND PROPRIETARY
-# 
-# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
 # Source Code License included in this distribution package. See LICENSE.
-# By accessing, using, copying or modifying this work you indicate your 
-# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 """
+Menu actions.
 """
 
 import sgtk
-from sgtk import TankError
-from sgtk.platform.qt import QtGui, QtCore
 
-class Action(object):
+
+class ActionBase(object):
```

Comment:
I'd avoid the term Menu item because it then implies we have a menu, which is implementation detail of the workfiles dialog.